README.md

```diff
@@ -3,17 +3,41 @@
+* finding samples by the observations they contain
+* finding samples by arbitrary metadata searches
+* summarizing samples over metadata
+* retieval of sample data into BIOM
+* discovering metadata categories
+* pulling out sample data from different processing types (e.g., search over 16S, retrieve WGS)
 
-Additionally, assuming a comprehensive set of metadata are available, you can obtain summarized environment information:
-    
-    $ redbiom search observations --context "greengenes-13_8-97" 367523 3064251 | redbiom summarize samples --from - --category "env_biome"
+Redbiom is designed to handle biological and technical replicates. Specifically, it allows for a one to many relationship between a sample's metadata and its data, both within and between preparation types.
+
+This repository defines the de facto redbiom data representation, and one possible interface into the resource. Other interfaces (e.g., Javascript) are possible to define. Please see the Design section below for details about how other interfaces can be written.
+
+By default, redbiom will search against `qiita.ucsd.edu`. This can be changed at runtime by setting the `REDBIOM_HOST` environmental variable, e.g., `export REDBIOM_HOST=http://qiita.ucsd.edu:7379`. The default host is **read-only** and administrative functions like loading data will not work against it.
+
+If you intend to **load** your own data, you must setup a local instance (please see the server installation instructions below). In addition, you must explicitly set the `REDBIOM_HOST` environment variable.
 
 # Installation
 
-Needs [Redis](https://redis.io/) (tested with 3.2.6) and [Webdis](http://webd.is/) (just clone the repo). Also depends on [BIOM](http://biom-format.org/) (tested on 2.1.5), [Pandas](http://pandas.pydata.org/) (tested on 0.19.0), [Click](http://click.pocoo.org/5/) (required >= 6.7) and [scipy](https://www.scipy.org/) (whatever BIOM is happy with). It is not necessary to have super user access.
+### General requirements
+
+Redbiom epends on [BIOM](http://biom-format.org/) (tested on >= 2.1.5), [Pandas](http://pandas.pydata.org/) (tested on 0.19.0), [Click](http://click.pocoo.org/5/) (required >= 6.7), [nltk](http://www.nltk.org/) (tested on 3.2.2), [joblib](https://pythonhosted.org/joblib/) (tested on 0.9.3), and [scipy](https://www.scipy.org/) (whatever BIOM is happy with).
+
+### Client
+
+If you would like to use redbiom as only a client (which is the general case), then the following instructions apply.
+
+    $ git clone https://github.com/wasade/redbiom.git
+    $ pip install -e .  
+
+### Server
+
+If you would like to run your own resource, and load data locally or private data, then the following instructions apply.
+
+In addition to the general requirements, redbiom server needs [Redis](https://redis.io/) (tested with 3.2.6) and [Webdis](http://webd.is/) (just clone the repo). It is not necessary to have super user access to establish a redbiom server.
```

Comment:
Should we add that we use 2.8.17 for actual travis testing and for deploy? Which means that arguably, the most tested is that version.